### PR TITLE
Add update file for wesnoth

### PR DESCRIPTION
**Pull request by hanspolo**

### CODE REVIEW

File: srcpkgs/wesnoth/update [hunk 1]
```diff
@@ -0,0 +1,3 @@
+pkgname=wesnoth
+version=1.12.2
```
Comment (Gottox) [inline srcpkgs/wesnoth/update]: above lines can be dropped

File: srcpkgs/wesnoth/update [hunk 2]
```diff
@@ -0,0 +1,3 @@
...
+ignore=1.13
```
Comment (Gottox) [inline srcpkgs/wesnoth/update]: missing \n at the end.

**Comment by xtraeme** [comment]
What will happen when 1.15 is available? ignoring just 1.13 is wrong.

**Comment by Gottox** [comment]
ignore="1.1[13679]"

**Pull request merged**